instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import './CommentSection.css';
+import Comment from './Comment';
+
+
+
+
+class CommentSection extends Component {
+    
+    constructor(){
+        super();
+        this.state={
+          comments: [],
+          inputText: ''
+        }
+      }
+
+      componentDidMount(){
+        this.setState({ comments: this.props.comments})
+      }
+
+      addNewComment = ev =>{
+        ev.preventDefault();
+        if(!(this.state.inputText === '')){
```

Comment:
Since an empty string is false-y, and any other string is truth-y, you could be tricky and skip the equality operator here.